### ISSUE TITLE
When I set"verticalDepth" option, I can't use drag&drop.

### ISSUE THREAD

**Issue by zxc7922927** [dabeng/OrgChart#165]
I just have same issue with that case: https://github.com/dabeng/OrgChart/issues/120
But he closed it

**Comment by dabeng** [comment]
yes, as the description in #120 , this feature is in progressing.

**Comment by dabeng** [comment]
Hi @zxc7922927 From the version 3.4.0, you can drag&drop in the hybrid(horizontal + vertical) chart with the help of [JSONDigger](https://github.com/dabeng/json-digger) utility class. I have created the [Demo](https://codepen.io/dabeng/pen/yLqQGRb) on codepen.io for you.
